examples/plot_simulate_somato.py

```diff
@@ -94,45 +143,99 @@
-    'evdist2', mu=82., sigma=0., numspikes=1, weights_ampa=weights_ampa_d,
-    weights_nmda=weights_nmda_d, location='distal',
-    synaptic_delays=synaptic_delays_d, seedcore=2)
-
-# proximal input occurs before distals
-weights_ampa_p = {'L2_basket': 0.003, 'L2_pyramidal': 0.0025,
-                  'L5_basket': 0.004, 'L5_pyramidal': 0.001}
-weights_nmda_p = {'L2_basket': 0.003, 'L5_basket': 0.004}
+    'evprox1', mu=21., sigma=4., numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_p, weights_nmda=weights_nmda_p,
+    location='proximal', synaptic_delays=synaptic_delays_p, seedcore=6)
+
+# Late proximal drive
+weights_ampa_p = {'L2_basket': 0.003, 'L2_pyramidal': 0.0039,
+                  'L5_basket': 0.004, 'L5_pyramidal': 0.0020}
+weights_nmda_p = {'L2_basket': 0.001, 'L2_pyramidal': 0.0005,
+                  'L5_basket': 0.002, 'L5_pyramidal': 0.0020}
 synaptic_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
                      'L5_basket': 1.0, 'L5_pyramidal': 1.0}
+
+net.add_evoked_drive(
+    'evprox2', mu=134., sigma=4.5, numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_p, weights_nmda=weights_nmda_p,
+    location='proximal', synaptic_delays=synaptic_delays_p, seedcore=6)
+
+# Early distal drive
+weights_ampa_d = {'L2_basket': 0.0043, 'L2_pyramidal': 0.0032,
+                  'L5_pyramidal': 0.0009}
+weights_nmda_d = {'L2_basket': 0.0029, 'L2_pyramidal': 0.0051,
+                  'L5_pyramidal': 0.0010}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+
+net.add_evoked_drive(
+    'evdist1', mu=32., sigma=2.5, numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=6)
+
+# Late distal drive
+weights_ampa_d = {'L2_basket': 0.0041, 'L2_pyramidal': 0.0019,
+                  'L5_pyramidal': 0.0018}
+weights_nmda_d = {'L2_basket': 0.0032, 'L2_pyramidal': 0.0018,
+                  'L5_pyramidal': 0.0017}
+synaptic_delays_d = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+
 net.add_evoked_drive(
-    'evprox1', mu=20.0, sigma=0., numspikes=1, weights_ampa=weights_ampa_p,
-    weights_nmda=weights_nmda_p, location='proximal',
-    synaptic_delays=synaptic_delays_p, seedcore=6)
+    'evdist2', mu=84., sigma=4.5, numspikes=1, sync_within_trial=True,
+    weights_ampa=weights_ampa_d, weights_nmda=weights_nmda_d,
+    location='distal', synaptic_delays=synaptic_delays_d, seedcore=2)
 
-dpl = simulate_dipole(net, n_trials=1)
+###############################################################################
+# Now we run the simulation over 2 trials so that we can plot the average
+# aggregate dipole. As in :ref:`the MPIBackend example
+# <sphx_glr_auto_examples_plot_simulate_mpi_backend.py>`, we can use
+# ``MPIBackend`` to reduce the simulation time by parallizing across cells in
```

Comment:
I was going to merge but then noticed there are still references to `MPIBackend` :)

Reply:
Good catch. Fixed in https://github.com/jonescompneurolab/hnn-core/pull/253/commits/1d6e6391894eb181947afc9c6c1b6b5a09bd915d.